haskell/sbp.cabal

```diff
@@ -1,5 +1,5 @@
 name:                  sbp
-version:               2.2.3
+version:               2.2.3-dirty
```

Comment:
@dt-exafore it means when you generated the code, there were uncommitted changes - this should get backed out.